subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/UpgradeDependencyPlugin.java

```diff
@@ -229,9 +232,9 @@ public void execute(final CreatePullRequestTask task) {
                 task.setGitHubApiUrl(conf.getGitHub().getApiUrl());
                 task.setDryRun(conf.isDryRun());
                 task.setAuthToken(conf.getLenient().getGitHub().getWriteAuthToken());
-                task.setVersionUpgrade(upgradeDependencyExtension);
-                task.setPullRequestTitle(getPullRequestTitle(task));
-                task.setPullRequestDescription(getPullRequestDescription(task));
+                task.setBaseBranch(upgradeDependencyExtension.getBaseBranch());
```

Comment:
deferred configuration?

Reply:
Yes good catch, in this case it's needed